cmd/kubelet/app/server.go

```diff
@@ -268,6 +272,8 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&s.ReallyCrashForTesting, "really-crash-for-testing", s.ReallyCrashForTesting, "If true, when panics occur crash. Intended for testing.")
 	fs.Float64Var(&s.ChaosChance, "chaos-chance", s.ChaosChance, "If > 0.0, introduce random client errors and latency. Intended for testing. [default=0.0]")
 	fs.BoolVar(&s.Containerized, "containerized", s.Containerized, "Experimental support for running kubelet in a container.  Intended for testing. [default=false]")
+	fs.BoolVar(&s.ReconcileCIDR, "reconcile-cidr", s.ReconcileCIDR, "Reconcile node CIDR with the CIDR specified by the API server. No-op if register-node or configure-cbr0 is false. [default=true]")
```

Comment:
So `configure-cbr0` tells the kubelet whether or not to create / configure cbr0 and `reconcile-cidr` tells the kubelet whether or not to *reconfigure* the bridge based on the cidr allocation from the apiserver. Is the reason that we don't want to reconfigure the bridge because the interaction of restarting docker + supervisord causes the cluster to be unhealthy? If we want to make the master node schedulable at some point then it will need to be able to reconfigure cbr0 when it gets a cidr assigned.